code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -404,7 +404,17 @@ var/global/list/activated_medevac_stretchers = list()
 
 //bedroll
 /obj/structure/bed/bedroll
-	name = "bedroll"
-	desc = "bedroll"
+	name = "unfolded bedroll"
+	desc = "Perfect for those long missions, when there's nowhere else to sleep, you remembered to bring at least one thing of comfort."
+	icon = 'icons/monkey_icos.dmi'
 	icon_state = "bedroll_o"
+	buckling_y = 0
+	foldabletype = /obj/item/roller/bedroll
+	accepts_bodybag = FALSE
+
+/obj/item/roller/bedroll
+	name = "folded bedroll"
+	desc = "A standard issue USCMC bedroll, They've been in service for as long as you can remember. the tag on it states to unfold it before rest, but who needs rules anyway, right?"
```

Comment:
You have a capital letter after a comma "bedroll, They've" and no capital after a full stop "remember. the tag".

Otherwise looks fine. Good PR 👍

Reply:
fixxed